src/app/(marketing)/profile/_components/OperatedGuildCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { CheckMark } from "@/components/CheckMark"
+import { Avatar, AvatarFallback } from "@/components/ui/Avatar"
+import { Card } from "@/components/ui/Card"
+import { Skeleton } from "@/components/ui/Skeleton"
+import { User } from "@phosphor-icons/react/dist/ssr"
+import { AvatarImage } from "@radix-ui/react-avatar"
+import Image from "next/image"
+
+export const OperatedGuildCard = () => {
+  return (
+    <Card className="flex">
+      <div className="relative w-1/3">
```

Comment:
I think it would be easier to use grid here with 3 cols. That would also help to easily handle the mobile view which is missing right now 😉